utils/constants.js

```diff
@@ -29,16 +29,23 @@ export const faq_data = [
   {
     que: "What are the future enhancements?",
     ans:
-      "Bulk upload future, MediaWiki OAuth integration and Direct uploading to Commons.",
+      "Bulk upload feature, direct upload to Wikimedia Commons and addition of newer libraries are some features being actively worked upon.",
   },
   {
     que: "Can I upload my own book?",
     ans:
-      "If an appropriate license is attached to the book which allows it to be archived(which doesn't allow copyright infringement suits, etc.), then book can be uploaded.",
+      "If an appropriate license is attached to the book which allows it to be archived (which doesn't allow copyright infringement suits, etc.), then book can be uploaded.",
   },
   {
     que: "Can I delete or undo an upload?",
     ans:
-      "Archives are supposed to be read-only and deleting is a 'write' operation. So it doesn't support.",
+      "Archives are supposed to be read-only and deleting is a 'write' operation. So it is not supported.",
   },
 ];
+
+export const header_links = [
+  { name: "Upload", href: "/", id: "form-tabs-1" },
+  { name: "Queue", href: "/queue", id: "form-tabs-2" },
+  { name: "Stats", href: "/stats", id: "form-tabs-3" },
+  { name: "FAQs", href: "/faqs", id: "form-tabs-4" },
```

Comment:
Remove "," from here.

Reply:
Sir each of this represents an Item in an array. If I remove this commas I will cause errors.
![Screenshot from 2023-10-10 21-38-33](https://github.com/coderwassananmol/BUB2/assets/88565289/2f3a68b7-a5f4-4d33-a9bb-c11c60254018)